js/widgets/forms/flipswitch.js

```diff
@@ -34,7 +34,7 @@ $.widget( "mobile.flipswitch", $.extend({
 					flipswitch: this.element.parent(),
 					on: this.element.find( ".ui-flipswitch-on" ).eq( 0 ),
 					off: this.element.find( ".ui-flipswitch-off" ).eq(0),
-					type: this.element.get( 0 ).tagName
+					type: this.element.get( 0 ).nodeName.toLowerCase()
```

Comment:
This should be moved out of the conditional it needs to run either way. Just do `this.type = this.element.get( 0 ).nodeName.toLowerCase()`  as the first thing in create and remove from `enhance`.